CONTRIBUTING.md

```diff
@@ -629,6 +629,10 @@ should be canceled.
 
 ## Approvers and Maintainers
 
+### Triagers
+
+- [Cheng-Zhen Yang](https://github.com/scorpionknifes), mx51
```

Comment:
```suggestion
- [Cheng-Zhen Yang](https://github.com/scorpionknifes)
```
I'm currently not company-affiliated and won't be anytime soon (also switching roles next month).

Is it possible to keep company empty? thanks.

Reply:
If you're working as an independant, how about marking it as such?

```suggestion
- [Cheng-Zhen Yang](https://github.com/scorpionknifes), Independant
```

Reply:
Thanks, that works